### PR TITLE
[skip ci] Updating version to 0.5

### PR DESCRIPTION
**Pull request by stuclem**
[skip ci]  Updating the doc version to 0.5. Fixes https://github.com/vmware/vic/issues/1964.

### CODE REVIEW

**Comment by stuclem** [comment]
Purely cosmetic changes, so can LGTM myself.

File: doc/user_doc/vic_admin/book.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-	"title": "VMWare vSphere Integrated Containers for vSphere Administrators, ONSITE BETA",
+	"title": "VMWare vSphere Integrated Containers for vSphere Administrators, version 0.5",
```
Comment (anchal-agrawal) [inline doc/user_doc/vic_admin/book.json]: Nit-picky, but it should be `VMware`.

File: doc/user_doc/vic_app_dev/book.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-	"title": "Developing Container Applications with VMWare vSphere Integrated Containers, ONSITE BETA",
+	"title": "Developing Container Applications with VMWare vSphere Integrated Containers, version 0.5",
```
Comment (anchal-agrawal) [inline doc/user_doc/vic_app_dev/book.json]: ditto

File: doc/user_doc/vic_installation/book.json [hunk 3]
```diff
@@ -1,5 +1,5 @@
 {
-	"title": "VMWare vSphere Integrated Containers Installation, ONSITE BETA",
+	"title": "VMWare vSphere Integrated Containers Installation, version 0.5",
```
Comment (anchal-agrawal) [inline doc/user_doc/vic_installation/book.json]: ditto

File: doc/user_doc/vic_admin/book.json (hunk 1 above)
Comment (stuclem) [inline doc/user_doc/vic_admin/book.json]: Not nit-picky. Good spot. Fixed all 3 instances.

**Comment by anchal-agrawal** [comment]
LGTM

**Comment by stuclem** [comment]
Thanks @anchal-agrawal!

**Pull request merged**